test/unit_test/engine/test_engine.py

```diff
@@ -321,17 +321,56 @@ def test_run_no_search(self, mock_local_system_init, tmp_path):
         assert output_node.model_config == onnx_model_config
         assert expected_metrics == output_node.metrics.value
 
-        output_model_dir = output_dir / "output_model"
```

Comment:
please update the tests in this too https://github.com/microsoft/Olive/blob/acc87632d11a7d9aa7117f50ccfbe2fec963189b/test/unit_test/cli/test_cli.py#L116

Reply:
https://github.com/microsoft/Olive/blob/acc87632d11a7d9aa7117f50ccfbe2fec963189b/olive/cli/base.py#L412

Reply:
curious why this test can pass

Reply:
the test passes since base.py was not updated and we just mock the run output. we might have to update one of the cli tests to not do the mock.